hollow/src/main/java/com/netflix/hollow/tools/diff/HollowDiffMatcher.java

```diff
@@ -99,9 +99,11 @@ public void calculateMatches() {
             try {
                 matchedOrdinal = fromIdx.getMatchingOrdinal(key);
             } catch(NullPointerException ex) {
-                LOGGER.error("Null values found for type " + toTypeState.getSchema().getName()
+                LOGGER.log(Level.SEVERE, "Null values found for type " + toTypeState.getSchema().getName()
+                        + " with key field values " + Arrays.asList(key) + " at ordinal : " + candidateToMatchOrdinal);
+                throw new RuntimeException("Error fetching matching ordinal for null type " + toTypeState.getSchema().getName()
                         + " with key field values " + Arrays.asList(key) + " at ordinal : " + candidateToMatchOrdinal
-                        + " with stacktrace : " + ex.getStackTrace());
+                        + "with stack trace " + ex.getCause());
```

Comment:
I'm not sure if `ex.getCause()` will do what you want - I think you probably want to pass `ex` _as_ the cause (as the second argument to `new RuntimeException`)

Reply:
fixed